project-base/storefront/components/Blocks/Banners/BannersSliderItem.tsx

```diff
@@ -16,9 +16,7 @@ export const BannersSliderItem: FC<BannersSliderItemProps> = ({
             {!image ? (
                 <BannerImage alt="no image" src="images/optimized-noimage.webp" />
             ) : (
-                <picture>
-                    <BannerImage alt={image.name || name} src={image.url} />
-                </picture>
+                <BannerImage alt={image.name || name} src={image.url} />
```

Comment:
Shouldn't we specify the width here? There were sizes (and additionalSizes) specified in the `images.yaml`

Reply:
ah, I did not notice it is not the Image component :+1: 

> But I have replaced this place and others where img has been used with out Image component. So now it's even better optimized. 

:ok_hand: